GreenhouseNode/GreenhouseNode/links/Temperature.c

```diff
@@ -0,0 +1,148 @@
+/*
+ * Temperature.c
+ *
+ * Created: 12/8/2021 9:50:07 PM
+ *  Author: tolde
+ */ 
+
+#include <ATMEGA_FreeRTOS.h>
+#include <semphr.h>
+#include <stdio.h>
+#include <rc_servo.h>
+#include <task.h>
+#include <hih8120.h>
+#include "PrintShared.h"
+#include "event_groups.h"
+#include "Temperature.h"
+
+EventGroupHandle_t startGroup_task;
+EventBits_t _readyBit;
+
+typedef struct tempHandler
+{
+	uint16_t humidity;
```

Comment:
Why is humidity in this class? I would expect `Temperature.c` to only contain temperature stuff. You should probably move all humidity things to a `humidityHandler.c` or something like that.